sgrep/bin/main_sgrep.ml

```diff
@@ -540,10 +566,14 @@ let main () =
     (* main entry *)
     (* --------------------------------------------------------- *)
     | x::xs -> 
-        if !rules_file <> ""
-        then sgrep_with_rules !rules_file (x::xs)
-        else sgrep_with_one_pattern (x::xs)
-
+        (try (
+          if !rules_file <> ""
+          then sgrep_with_rules !rules_file (x::xs)
+          else sgrep_with_one_pattern (x::xs)
+        ) with exn -> 
+          pr (format_output_exception exn); (* todo, should be pr2 probably *)
```

Comment:
pr means on stdout, pr2 means on stderr.
I don't know what sgrep-lint captures, I think stdout, so probably good to have pr